### DISCUSSION TITLE
"Show more" but no "show less" button?

### DISCUSSION THREAD

**Discussion by dietmar-schoder** [scalar/scalar#3442] [Q&A]
I made it run in a .net 8 Web Application, but when I click on the "Show more" button for any endpoint group I can never collapse the expanded block anymore, because - as far as I can see - no "Show less" button appears.

**Comment by marclave** [comment]
interesting! @cameronrohani what do you think about this?

**Comment by dietmar-schoder** [comment]
This is what it looks like:
![image](https://github.com/user-attachments/assets/2d0c74b7-33c8-4a92-85d9-fd51c350299b)

![image](https://github.com/user-attachments/assets/79583a88-6ca8-429b-98fb-66833fd95df2)

Especially when I set this option to "false", I cannot "show less" anymore any section I "showed more".
![image](https://github.com/user-attachments/assets/59647d90-556d-4301-ab0b-e0ab71fb8fdb)
